_posts/2019-02-02-opensourcedesign-fosdem.md

```diff
@@ -52,7 +52,7 @@ By submitting a proposal you consent to be recorded and agree to license the con
 
 #### Submission deadline:
 
-1st December 2018
+8th December 2018
```

Comment:
You could have made it like: 
--1st December 2018-- (Extended: 8th December 2018)

Reply:
good point. i'll update it now.

👍

Reply:
used this GH syntax: https://webapps.stackexchange.com/questions/14986/strikethrough-with-github-markdown